docs/gecko-profile-format.md

```diff
@@ -8,17 +8,14 @@ The Gecko Profiler can be run on its own by executing the following code from a
 
 ```js
 (() => {
-  const profiler = Components.classes["@mozilla.org/tools/profiler;1"]
-    .getService(Components.interfaces.nsIProfiler);
```

Comment:
mmm don't we have to import `Services` though ? Here is what I use in a test:
```javascript
ChromeUtils.import("resource://gre/modules/Services.jsm");
```

Reply:
Well, it depends on how one uses this snippet. In the console for example, which was my use case, you don't need to import anything. In some environments, you may want to use Components, Cu or ChromeUtils as in your test. For example code in documentation my preference is to have as concise a snippet as possible, but your mileage may vary.

Reply:
hey @gregtatum, do you have an opinion about this?

Reply:
I'm fine with the simpler code.